config-connector/solutions/iam/kpt/pubsub-topic/README.md

```diff
@@ -0,0 +1,39 @@
+Pub/Sub Topic
+==================================================
+# NAME
+  pubsub-topic
+# SYNOPSIS
+  Config Connector compatible YAML files to grant a role to a particular IAM member for a PubSub topic.
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/pubsub-topic .
+  ```
+# REQUIREMENTS
+  A Config Connector installation managing a GCP project with the Pub/Sub API enabled.
+# USAGE
+## SETTERS
+|    NAME    |        VALUE        |     SET BY      |         DESCRIPTION          | COUNT |
+|------------|---------------------|-----------------|------------------------------|-------|
+| iam-member | ${IAM_MEMBER?}      | PLACEHOLDER     | identity to grant privileges | 1     |
+| role       | roles/pubsub.editor | package-default | IAM role to grant            | 1     |
+| topic-name | allowed-topic       | package-default | name of PubSub topic         | 2     |
+
+  Replace `${IAM_MEMBER?}` with the GCP identity to grant access to.
+  ```
+  kpt cfg set . iam-member user:name@example.com
```

Comment:
in the command, should we put `${IAM_MEMBER?}`, not `iam-member`, so it is clear what we are replacing.

Reply:
iam-member is the name of the setter and ${IAM_MEMBER?} is the placeholder value it starts with.

Reply:
Do you need any follow up from @AlexBulankou? If not, could you resolve the comment?

Reply:
Got it, sounds good.